functions/Get-DbaStartupParameter.ps1

```diff
@@ -0,0 +1,163 @@
+
+.DESCRIPTION
+Displays values for a detailed list of SQL Server Startup Parameters including Master Data Path, Master Log path, Error Log, Trace Flags, Parameter String and much more.
+
+This command relies on remote Windows Server (SQL WMI/WinRm) access. You can pass alternative Windows credentials by using the -Credential parameter. 
+	
+See https://msdn.microsoft.com/en-us/library/ms190737.aspx for more information.
+	
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the Windows Server as a different Windows user. 
+	
+.PARAMETER Simple
+Shows a simplified output including only Server, Master Data Path, Master Log path, ErrorLog, TraceFlags and ParameterString
+
+.NOTES
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+.LINK
+ https://dbatools.io/Get-DbaStartupParameter
+
+.EXAMPLE
+Get-DbaStartupParameter -SqlServer sql2014
+
+Logs into SQL WMI as the current user then displays the values for numerous startup parameters.
+
+.EXAMPLE
+$wincred = Get-Credential ad\sqladmin
+Get-DbaStartupParameter -SqlServer sql2014 -Credential $wincred -Simple
+
+Logs in to WMI using the ad\sqladmin credential and gathers simplified information about the SQL Server Startup Parameters.
+	
+#>	
+	[CmdletBinding()]
+	param ([parameter(ValueFromPipeline, Mandatory = $true)]
+		[Alias("ServerInstance", "SqlInstance")]
+		[object[]]$SqlServer,
+		[Alias("SqlCredential")]
+		[PSCredential]$Credential,
+		[switch]$Simple
+	)
+	
+	PROCESS
+	{
+		foreach ($servername in $SqlServer)
+		{
+			$servercount++
+			try
+			{
+				$instancename = ($servername.Split('\'))[1]
+				Write-Verbose "Attempting to connect to $servername"
+				
+				if ($instancename.Length -eq 0) { $instancename = "MSSQLSERVER" }
+				
+				$displayname = "SQL Server ($instancename)"
+				
+				$Scriptblock = {
+					$servername = $args[0]
+					$displayname = $args[1]
+					
+					$wmisvc = $wmi.Services | Where-Object { $_.DisplayName -eq $displayname }
+					
+					$params = $wmisvc.StartupParameters -split ';'
+					
+					$masterdata = $params | Where-Object { $_.StartsWith('-d') }
+					$masterlog = $params | Where-Object { $_.StartsWith('-l') }
+					$errorlog = $params | Where-Object { $_.StartsWith('-e') }
+					$traceflags = $params | Where-Object { $_.StartsWith('-T') }
```

Comment:
We should look for the lowercase trace flags that might have been set by accident or during a support case with Microsoft and then left on. 

msdn.microsoft.com/en-us/library/ms190737.aspx

_** Important ** When specifying a trace flag with the -T option, use an uppercase "T" to pass the trace flag number. A lowercase "t" is accepted by SQL Server, but this sets other internal trace flags that are required only by SQL Server support engineers. (Parameters specified in the Control Panel startup window are not read.)_